src/chartkit/components/Pie.js

```diff
@@ -19,17 +19,40 @@ const PieTitle = styled('div')`
   margin-bottom: 5px;
 `;
 
+const StyleTooltip = styled('Tooltip')`
```

Comment:
I don't think the style is applying, we'll need `className` passed to the DOM element in the `Tooltip` component for it to apply. Also I think it should be `styled(Tooltip)` (`Tooltip` as component reference, not string)

Reply:
@hlminh2000 Works for me using the component name as a string.

Reply:
<img width="993" alt="screen shot 2019-02-04 at 11 53 44 am" src="https://user-images.githubusercontent.com/3919995/52223463-ac628a80-2873-11e9-9c73-b6eae32b6d87.png">